website/docs/guides/1.1.0-upgrade-guide.html.markdown

```diff
@@ -8,73 +8,90 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
 
 # MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
 
-The Terraform MongoDB Atlas Provider version 1.0.0 has a number of new and exciting features and changes.
+The Terraform MongoDB Atlas Provider version 1.1.0 has a number of new and exciting features and changes.
 
-* Cloud Backup Snapshot replaces the now deprecated Cloud Provider Snapshot.
-* Cloud Backup Snapshot Restore Job replaces the now deprecated Cloud Provider Snapshot Restore Job.
+New Features:
 
-1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
+* You can now manage multi-cloud deployments with [`mongodbatlas_advanced_clusters`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_clusters).
```

Comment:
These links will work when this version is released, correct?

Reply:
Correct, they won't work till the release goes out.